saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -308,3 +312,245 @@ def perform_mutation(cls, _root, info, **data):
         fulfillment.refresh_from_db(fields=["status"])
         order.refresh_from_db(fields=["status"])
         return FulfillmentCancel(fulfillment=fulfillment, order=order)
+
+
+class OrderRefundLineInput(graphene.InputObjectType):
+    order_line_id = graphene.ID(
```

Comment:
IMO there is no need to add `id` suffix. In our code base we are using just name eg order_line. We should be consistent according to our codebase.

Reply:
I was adding them based on the `OrderFulfillLineInput`, which uses `order_line_id` as an identifiactor.

Reply:
@d-wysocki As i'm looking into codebase it seems we use the `_id` prefix in quite a few places if not mostly (search `_id = graphene.ID`) I'm a bit confused what the pattern is.